Src/VimCore/Constants.fs

```diff
@@ -36,12 +36,7 @@ module VimConstants =
     [<Literal>]
     let MainKeyProcessorName = "VsVim";
 
-#if DEBUG
-    [<Literal>]
-    let VersionNumber = "2.6.99.99 Debug"
-#else

```

Comment:
I have been grateful for this feature, which prevents Visual Studio from constantly badgering me that my VsVim extension has a newer version. How does that conflict with the desire for versioned assemblies?

Reply:
Mostly it just added extra pain in the infrastructure to set it up properly. I didn't think anyone was still using this (originally designed it for a different purpose). Let me restore this.

Reply:
How do you typically install VsVim? After looking deeper at this I don't think this is related to updates. This should only really control the value displayed in Help About. The actual version of the VSIX is fixed in source.extension.manifest and that is what VS should be triggering update requests on. 

If you install from vsixgallery.com the versioning story is a bit different.

Reply:
I have my own master in my fork which I periodically reset to your upstream master, merge my various feature branches, build, and then xcopy deploy into VS2017. My experience with the nagging is maybe from other extensions. If the the manifest is upgraded to 2.7.0.0, that should stop the nagging unrelated to this version (which I see from the `:ver` command).

Yeah, so don't worry about it.

Reply:
Gotcha. As long as we keep the version in master ahead of release then your scenario should continue to work. I have a similar process for updating which is why I haven't seen this in a while either.

Reply:
That reminds me. Need to get the version in master moved to 2.7.